### PR TITLE
devdoston/favorites entity added

### PR DESCRIPTION
**Pull request by devdoston**

### CODE REVIEW

File: FurnitureShop/src/FurnitureShop.Data/Entities/FavoritesProduct.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    public Guid ProductId { get; set; }
+
+    [ForeignKey(nameof(ProductId))]
+    public virtual Product? Product { get; set; }
+
+    public Guid FavoritesId { get; set; }
+
+    [ForeignKey(nameof(FavoritesId))]
+    public virtual Favorites? Favorites { get; set; }
+}
```
Comment (javlonfattoev) [inline FurnitureShop/src/FurnitureShop.Data/Entities/FavoritesProduct.cs:17]: ortiqcha entity bor
```C#
public class FavoritesProduct
{
    public Guid Id { get; set; }

    [Key]
    public Guid UserId { get; set; }

    [ForeignKey(nameof(UserId))]
    public virtual AppUser? User { get; set; }
    public Guid ProductId { get; set; }
    [ForeignKey(nameof(ProductId))]
    public virtual Product? Product { get; set; }
}
```

Comment (javlonfattoev) [reply FurnitureShop/src/FurnitureShop.Data/Entities/FavoritesProduct.cs:17]: shuni ozi yetarli

**Review by javlonfattoev — CHANGES_REQUESTED** [review]

File: FurnitureShop/src/FurnitureShop.Data/Entities/FavoritesProduct.cs (hunk 1 above)
Comment (devdoston) [reply FurnitureShop/src/FurnitureShop.Data/Entities/FavoritesProduct.cs:17]: ustoz favorites ni ozi kerak emas shunday deyapsizmi

Comment (javlonfattoev) [reply FurnitureShop/src/FurnitureShop.Data/Entities/FavoritesProduct.cs:17]: ha

**Pull request closed without merging**